app/build.gradle

```diff
@@ -60,10 +60,10 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.6.1'
     implementation 'com.google.android.material:material:1.11.0'
 
-    implementation platform("androidx.compose:compose-bom:$compose_bom_version")
-    implementation "androidx.compose.ui:ui"
-    implementation "androidx.compose.material:material"
-    implementation "androidx.compose.ui:ui-tooling-preview"
+    //noinspection BomWithoutPlatform
```

Comment:
I think this is only needed if we using a Bom: https://googlesamples.github.io/android-custom-lint-rules/checks/BomWithoutPlatform.md.html

Reply:
yes 🤦 this is from a intermediate modification, I forgot to remove it before pushing, thanks!

Reply:
removed 886a08dad8981b61e2ca8cbaf7ac8c0b733c56f6